src/content/docs/apm/agents/go-agent/configuration/go-agent-code-level-metrics-config.mdx

```diff
@@ -126,6 +126,57 @@ There are additional configuration options you may wish to use to further refine
 
   * It is also possible to directly set the `CodeLevelMetrics.IgnoredPrefixes` member of the `Config` struct (which is a `[]string` value), but we recommend using one of the above-mentioned methods instead of directly manipulating the `Config` structure.
 
+<Callout variant="important">
+In Go Agent versions prior to 3.20.0, this option was named with in the singular form rather than plural (i.e., `ConfigCodeLevelMetricsIgnoredPrefix`). These names are deprecated and the ones documented here should be used. The older names are also still supported for backward compatibility.
+</Callout>
+
+  </Collapser>
+  <Collapser
+    id="go-clm-config-redact-ignored-prefix"
+    title={<>Redact ignored module prefixes</>}
+  >
+
+  If, for reasons such as confidentiality, you choose to ignore some modules via
+  `ConfigCodeLevelMetricsIgnoredPrefixes` option, you may also wish to redact the list of such prefixes
+  from the agent's reported configuration data as well.
+
+  This is accomplished by setting the `ConfigCodeLevelMetricsRedactIgnoredPrefixes` option. If given a `true` value,
+  your list of ignored prefixes will not be shown in the configuration data that are reported by the agent. Otherwise,
+  they will be reported.
+
+  Do one of the following:
+
+  * If configuring via the `NewApplication` function, add a `ConfigCodeLevelMetricsRedactIgnoredPrefixes` option:
+
+    ```go
+    app, err := newrelic.NewApplication(
+       newrelic.ConfigAppName("Your Application Name"),
+       newrelic.ConfigLicense("YOUR_NEW_RELIC_LICENSE_KEY"),
+       newrelic.ConfigCodeLevelMetricsEnabled(true),
+       newrelic.ConfigCodeLevelMetricsRedactIgnoredPrefixes(true),
+    )
+    ```
+
+  * If configuring via environment variables, set `NEW_RELIC_CODE_LEVEL_METRICS_REDACT_IGNORED_PREFIXES`:
+    ```
+    NEW_RELIC_CODE_LEVEL_METRICS_REDACT_IGNORED_PREFIXES=true
+    ```
+
+    Don't forget to also include the `newrelic.ConfigFromEnvironment` option when setting up the application:
+
+    ```go
+    app, err := newrelic.NewApplication(
+       newrelic.ConfigAppName("Your Application Name"),
+       newrelic.ConfigLicense("YOUR_NEW_RELIC_LICENSE_KEY"),
+       newrelic.ConfigCodeLevelMetricsEnabled(true),
+       newrelic.ConfigFromEnvironment(),
+    )
+    ```
+
+<Callout variant="important">
+This redaction option is available in Go Agent versions 3.20.0 and later.
+</Callout>
+  </Collapser>
```

Comment:
This `</Collapser>` needs to be deleted and just be a blank line, the closing tag below will take care of this one!